src/index.html

```diff
@@ -1,11 +1,24 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Wbpack Exercise</title>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Leaderboard</title>
 </head>
 <body>
-    <h1>Hello webpack!</h1>
+  <h1 class="header">Leaderboard</h1>
+  <main class="container">
+    <div class="score-container">
+      <h3>Recent scores</h3>
+      <button type="submit" id="refresh">Refresh</button>
+      <ul class="scores-list"></ul>
+    </div>
+    <div class="input-container">
+      <h3>Add your score</h3>
+      <input type="text" id="name" value="name" placeholder="Your name">
+      <input type="text" id="score" value="score" placeholder="Your score">
+      <button type="submit" id="submit">Submit</button>
```

Comment:
- [ ] Kindly make sure the ` submit button` is smaller and looks as close as possible to the provided design

WireFrame Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/181936343-dbeddce0-f54d-49f6-af96-b5602860cc2e.png)|  ![image](https://user-images.githubusercontent.com/74814780/181936348-e059541f-a5fe-4d20-8f52-0dd815a626a8.png)